src/main/java/graphql/execution/instrumentation/dataloader/PerLevelDataLoaderDispatchStrategy.java

```diff
@@ -44,16 +48,40 @@ public class PerLevelDataLoaderDispatchStrategy implements DataLoaderDispatchStr
 
     static final long DEFAULT_BATCH_WINDOW_NANO_SECONDS_DEFAULT = 500_000L;
 
+    private final Map<DeferredCallContext, CallStack> callStackMap = new ConcurrentHashMap<>();
```

Comment:
Maybe rename this to `deferredCallStackMap` to make it clear that this field is only relevant for executions involving deferred data?